enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,325 @@
+---
+title: automated-backups-no-config
+authors:
+  - "@elbehery"
+reviewers:
+  - "@JoelSpeed"
+  - "@dusk125"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@vrutkovs"
+approvers:
+  - "@hasbro17"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-06-17
+last-updated: 2024-10-21
+tracking-link:
+  - https://issues.redhat.com/browse/ETCD-609
+see-also:
+  - "https://issues.redhat.com/browse/OCPSTRAT-1411"
+  - "https://issues.redhat.com/browse/OCPSTRAT-529"
+---
+
+
+# Automated Backups of etcd with No Config
+
+## Summary
+
+To enable automated backups of Etcd database and cluster resources of an Openshift cluster from day one after installation, 
+without provided configuration from the user.
+
+This document outlines the possible approaches that were investigated, discussed and tested, in addition to the final implemented approach.  
+
+## Motivation
+
+The current method of **Backup** and **Restore** of an Openshift cluster is outlined in  [Automated Backups of etcd](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md). 
+This approach relies on user supplied configuration, in the form of `config.openshift.io/v1alpha1 Backup` CR. This method utilizes `batchv1.CronJob` in order to take Backups of Etcd and cluster resources.
+
+To improve the user's experience, this work proposes taking etcd backup using default configurations from day one, without user interference.
+
+Moreover, for recovery in disaster scenarios, where more than one control-plane node is lost from an Openshift cluster, this proposal attempts to store backups on all control-plane nodes.
+Having backups of an Openshift cluster on all control-plane nodes, improves the recovery probability in disaster scenarios.
+
+### User Stories
+
+* As a cluster administrator, I would like an OCP cluster backups to be taken without user's supplied configuration from day one, after installation.
+* As a cluster administrator, I would like to an OCP cluster backups available on all master nodes, so that I have a recent cluster state to recover from in the event of losing control-plane nodes.
+
+### Goals
+
+* Backups should be taken without configuration after cluster installation from day 1.
+* Backups are saved to a default location on each master node's file system.
+* Backups are taken according to a default schedule.
+* Backups are being maintained according to a default retention policy.
+
+### Non-Goals
+
+* Save cluster backups to remote cloud storage (e.g. S3 Bucket).
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
```

Comment:
[Our current restore docs][1] list:

> A healthy control plane host to use as the recovery host.

as a prerequisite.  Possibly having backups on out-of-cluster storage is useful for audit purposes.  And it may be useful for some future recover-from-nothing procedure.  But at the moment, I don't see how it would be critical for Disaster Recovery.  Which is good, because if a non-goal was critical, that would make it pretty unclear why it was worth doing the enhancement at all ;)  Can we drop this line?

[1]: https://docs.openshift.com/container-platform/4.17/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

Reply:
`lgtm` 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽

Reply:
we already have this functionality discussed in the other EP, hence it's out of scope in here